projects/multiple-network-demo/package.json

```diff
@@ -12,9 +12,9 @@
   ],
   "scripts": {
     "deep-clean": "yarn clean && rm -rf node_modules",
-    "clean": "rm -rf dist",
-    "typecheck": "tsc --noEmit",
-    "typecheck:watch": "tsc --noEmit --watch",
+    "clean": "rm -rf dist tsconfig.tsbuildinfo",
+    "typecheck": "tsc -b",
```

Comment:
by removing -noEmit then "typecheck" command also compiles the code. We are using parcel for that

Reply:
See last comment